### PR TITLE
Proposed editorial and minor changes

### PR DESCRIPTION
**Pull request by lpiron**

### CODE REVIEW

File: 10-General.inc.md [hunk 1]
```diff
@@ -144,7 +146,7 @@ Note: While DASH only requires the presence of `moof/pssh` in the first CMAF chu
 
 A DASH client needs to recognize encrypted content and activate a suitable [=DRM system=], configuring it to decrypt content. The MPD informs a DASH client of the [=protection scheme=] used to protect content, identifies the [=content keys=] that are used and optionally provides the default [=DRM system configuration=] for a set of [=DRM systems=].
 
-A <dfn>DRM system configuration</dfn> is the complete data set required for a DASH client to activate a single [=DRM system=] and configure it to decrypt content using a single [=content key=]. <b>It is supplied by a combination of XML elements in the MPD and/or [=solution-specific logic and configuration=]</b>. A [=DRM system configuration=] often contains:
+The <dfn>DRM system configuration</dfn> is the complete data set required for a DASH client to activate a single [=DRM system=] and configure it to decrypt content using a single [=content key=]. It is supplied by a combination of XML elements in the MPD and/or [=solution-specific logic and configuration=]. The [=DRM system configuration=] often contains:
```
Comment (sandersaares) [inline 10-General.inc.md]: The emphasis here - now removed - was quite valuable, I think.

File: 10-General.inc.md [hunk 2]
```diff
@@ -161,9 +163,9 @@ Note: In theory, it is possible for the [=DRM system=] initialization data to be
...
+The presence of a `ContentProtection` descriptor with `schemeIdUri="urn:mpeg:dash:mp4protection:2011"` on an adaptation set informs a DASH client that all representations in the adaptation set are encrypted in conformance to Common Encryption ([[!DASH]] sections 5.8.4.1 and 5.8.5.2 and [[!CENC]] section 11) and require a [=DRM system=] to provide access.
 
-This descriptor is present for all encrypted content ([[!DASH]] 5.8.4.1). It SHALL be defined on the adaptation set level. The `value` attribute SHALL be either `cenc` or `cbcs`, matching the used [=protection scheme=]. The `cenc:default_KID` attribute SHALL be present and have a value matching the `default_KID` in the `tenc` box. The value SHALL be expressed in lowercase UUID string notation.
+This descriptor is present for all encrypted content ([[!DASH]] section 5.8.4.1). It SHALL be defined on the adaptation set level. The `value` attribute shall be either `cenc` or `cbcs`, matching the used [=protection scheme=] ([[!DASH]] section 5.8.5.2). The `cenc:default_KID` attribute SHALL be present and have a value matching the `default_KID` in the `tenc` box. The value SHALL be expressed in lowercase UUID string notation.
```
Comment (sandersaares) [inline 10-General.inc.md]: I think this SHALL should remain in all caps. Why did it become lowercase? Is the intent to remove this requirement?

File: 10-General.inc.md [hunk 3]
```diff
@@ -101,9 +103,9 @@ The structure of content protection related information in the CMAF containers u
...
 
-Initialization segments SHOULD NOT contain any `moov/pssh` box ([[!CMAF]] 7.4.3) and DASH clients MAY ignore such boxes when encountered. Instead, `pssh` boxes required for [=DRM system=] initialization are part of the [=DRM system configuration=] and SHOULD be placed in the MPD as `cenc:pssh` elements in [=DRM system=] specific `ContentProtection` descriptors.
+Initialization segments SHOULD NOT contain any `moov/pssh` box ([[!CMAF]], section 7.4.3) and DASH clients MAY ignore such boxes when encountered. Instead, `pssh` boxes required for [=DRM system=] initialization are part of the [=DRM system configuration=] and SHOULD be placed in the MPD as `cenc:pssh` elements in [=DRM system=] specific `ContentProtection` descriptors.
 
-Note: Placing the `pssh` boxes in the MPD has become common for purposes of operational agility - it is often easier to update MPD files than rewrite initialization segments when the default [=DRM system configuration=] needs to be updated. Furthermore, in some scenarios the appropriate set of `pssh` boxes is not known when the initialization segment is created.
```
Comment (sandersaares) [inline 10-General.inc.md:106]: Nit: There are several places here where a dash is replaced with a comma. I would say this change makes the sentence structure less natural, at least to me.

File: 40-LicenseRequestModel.inc.md [hunk 4]
```diff
@@ -14,17 +14,15 @@ This license request model defines a mechanism for achieving both goals. This re
 * DASH clients can execute DRM workflows without [=solution-specific logic and configuration=].
 * Custom code specific to a license server implementation is limited to backend business logic.
 
-These benefits increase in value with the size of the solution, as they reduce the development cost required to offer playback of encrypted content on a wide range of DRM-capable client platforms using different [=DRM systems=], with [=licenses=] potentially served by different license server implementations.
```
Comment (sandersaares) [inline 40-LicenseRequestModel.inc.md]: That's an important value statement explaining why the interoperable model has relevance. I think it adds value to the text. Perhaps we could further emphasize the affordances it gives to backend systems simplification but as a first step, this statement seemed worth having.

File: 10-General.inc.md (hunk 1 above)
Comment (lpiron) [reply 10-General.inc.md]: Ok to keep it

File: 10-General.inc.md (hunk 2 above)
Comment (lpiron) [reply 10-General.inc.md]: Change to "the value has to be matching the used protection scheme ([[!DASH]] section 5.8.5.2).

File: 10-General.inc.md (hunk 3 above)
Comment (lpiron) [reply 10-General.inc.md:106]: Keep dash instead of comma

File: 40-LicenseRequestModel.inc.md (hunk 4 above)
Comment (lpiron) [reply 40-LicenseRequestModel.inc.md]: Keep it.

**Pull request merged**